source/index.html

```diff
@@ -1,12 +1,89 @@
 <!DOCTYPE html>
 <html lang="ru">
+
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
 
+  <body>
+    <header class="header">
+      <a class="header__logo" href="">
+        <img class="header__logo-image" src="https://via.placeholder.com/191x39" width="191" height="39"
+          alt="Логотип Кэт энерджи">
+      </a>
+      <nav class="nav">
+        <button class="nav__toggle">Открыть меню</button>
+        <div class="nav__wrapper">
+          <ul class="nav__menu site-menu">
+            <li class="site-menu__item site-menu__item--active">
+              <a>Главная</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="#">Каталог продукции</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="form.html">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <h1 class="visually-hidden">Кэт энерджи</h1>
+      <section class="intro">
+        <header class="intro__header">
+          <img class="intro__bg-image" src="https://via.placeholder.com/320x305" width="320" height="305"
+            alt="Изображение кота">
+          <div class="intro__wrapper">
+            <h2 class="visually-hidden">Рекламный блок Кэт энерджи</h2>
+            <b class="intro__description">Функциональное питание для котов</b>
+            <p class="intro__slogan">Занялся собой? Займись котом!</p>
+            <img class="intro__promo-image" src="https://via.placeholder.com/280x270" width="280" height="270"
+              alt="Изображение банки кошачьего питания Кэт энерджи">
+            <a href="form.html" class="button intro__button">Подобрать программу</a>
+          </div>
+        </header>
+      </section>
+      <section class="programs">
+        <div class="programs__wrapper">
+          <h2 class="visually-hidden">Наши программы</h2>
+          <ul class="programs__list">
+            <li class="programs__item programs__item--slim">
+              <h3 class="programs__title">Похудение</h3>
```

Comment:
```suggestion
              <h3 class="programs__item-title">Похудение</h3>
```
Смотри, у тебя есть программы, и их тайтл, а еще есть каждая программа, и у неё есть заголовок, описание ссылка. Для того чтобы улучшить читабельность, предлагаю точно указывать чей это тайтл, дескрипшен, и линк.